docs/commands/cli-v2_product-release_list.md

```diff
@@ -0,0 +1,41 @@
+## cli-v2 product-release list
+
+List all the pipelines
```

Comment:
should be "product releases"

also - explicitly mention how many results are returned by default (when no start/end date are entered.